data/strings/strings.txt

```diff
@@ -22369,7 +22373,9 @@
     mr = %d ट्रॅक
     nb = %d stier
     nl = %d tracks
-    pl = %d tras
+    pl:one = %d trasa
+    pl:few = %d trasy
+    pl:many = %d tras
```

Comment:
```suggestion
    pl:few = %d trasy
    pl:many = %d tras
    pl:one = %d trasa
```

Our strings validator requires strings to be sorted alphabetically.

Reply:
Changed order as you showed